pkgs/stdenv/generic/setup.sh

```diff
@@ -422,50 +421,52 @@ fi
 
 
 substitute() {
-    local input="$1"
-    local output="$2"
+    local input=$1; shift
+    local output=$1; shift
```

Comment:
I short of like the "one variable, one shift" correspondence, but I'd be happy to change it.